servicetalk-http-api/src/main/java/io/servicetalk/http/api/UnsupportedHttpChunkException.java

```diff
@@ -15,11 +15,9 @@
  */
 package io.servicetalk.http.api;
 
-import io.servicetalk.serializer.api.SerializationException;
-
 import javax.annotation.Nullable;
 
-final class UnsupportedHttpChunkException extends SerializationException {
+final class UnsupportedHttpChunkException extends IllegalArgumentException {
```

Comment:
The change from `IllegalArgumentException` to `SerializationException` was made only in the main branch in #1673. This is not related to serialization. Reverting back to `IllegalArgumentException`.
cc @Scottmitch